### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by thunderclap82** [Fallenbagel/jellyseerr#167]
### Description

I would love an option for a Newsletter like Tautulli and Ombi have, that at a scheduled time an email blast goes out to registered users letting them know all the new content added since the previous week.

### Desired Behavior

Weekly email at scheduled time to showcase the newest media added to Emby/Jellyfin.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by gaming09** [comment]
yes!

**Comment by BLACK4585** [comment]
In case you are still searching, here is a plugin for Jellyfin that works well.
https://github.com/Cloud9Developer/Jellyfin-Newsletter-Plugin

**Comment by bonswouar** [comment]
> In case you are still searching, here is a plugin for Jellyfin that works well. https://github.com/Cloud9Developer/Jellyfin-Newsletter-Plugin

A JellySeerr-like design ready would be much better imo, with the email notifications, metadata integration and episode sync already working great.
(plus this plugin doesn't seem much maintained any more)

**Comment by BLACK4585** [comment]
That's why I wrote that it works, at least atm.
The default design is a bit boring, I changed it a bit to look better. Maybe you want to use this until Jellyseerr has a newsletter.
Body:
```
<html>
    <div>
        <table style='margin-left: auto; margin-right: auto;'>
            <tr> 
                <td width='100%' height='100%' style='vertical-align: top; background: linear-gradient(to bottom right, #3b2344, #043a4c);'> 
                    <table id='InsertHere' name='MainTable' style='margin-left: auto; margin-right: auto; border-spacing: 0 20px; padding-left: 2%; padding-right: 2%; padding-bottom: 1%;'> 
                        <tr style='text-align: center;'> 
                            <td colspan='2'> 
                                <span>
                                    <h1 id='Title' style='color:#FFFFFF;'><YOUR NAME> Jellyfin Newsletter</h1>
                                    <h3 id='Date' style='color:#FFFFFF;'>2024</h3>
                                </span> 
                            </td> 
                        </tr> 
                        <!-- Fill this in from code --> 
                        {EntryData}
                        <!-- Fill that in from code --> 
                    </table> 
                    <br>
                    <br>
                    <span style="text-align: center; display: block;">
                        <a href="<YOUR DOMAIN>" style="border: 2px solid white; border-radius: 15px; padding: 10px; background: linear-gradient(to bottom right, #713a91, #0785b6); color: white; text-decoration: none;"><SOME TEXT></a>
                    </span>
                    <br>
                    <br>
                </td> 
            </tr> 
        </table> 
    </div> 
</html>
```

Entry Data:
```
<tr class='boxed' style='outline: 2px solid #D3D3D3; background-color: #1f1f1f; border-radius: 20px; overflow: hidden;'>
                            <td class='lefttable' style='padding-right: 5%; padding-left: 2%; padding-top: 2%; padding-bottom: 2%; '>
                                <img style='width: 200px; height: 300px; border-radius: 20px;' src='{ImageURL}'>
                            </td>
                            <td class='righttable' style='vertical-align: top; padding-left: 5%; padding-right: 2%; padding-top: 2%; padding-bottom: 2%;'>
                                <p>
                                    <div style="border: 2px solid white; border-radius: 20px; padding: 15px;">
                                        <div id='SeriesTitle' class='text' style='color: #FFFFFF; text-align: center;'><h3>{Title}</h3></div>
                                        <div id='SeasonEpisode' class='text' style='color: #FFFFFF;'>{SeasonEpsInfo}</div>
                                        <hr>
                                        <div id='Description' class='text' style='color: #FFFFFF;'>{SeriesOverview}</div>
                                    </div>
                                </p>
                            </td>
                        </tr>
```
Don't forget to insert your custom data in the body.

**Comment by sunjam** [comment]
> That's why I wrote that it works, at least atm. The default design is a bit boring, I changed it a bit to look better. Maybe you want to use this until Jellyseerr has a newsletter. Body:
> 
> ```
> <html>
>     <div>
>         <table style='margin-left: auto; margin-right: auto;'>
>             <tr> 
>                 <td width='100%' height='100%' style='vertical-align: top; background: linear-gradient(to bottom right, #3b2344, #043a4c);'> 
>                     <table id='InsertHere' name='MainTable' style='margin-left: auto; margin-right: auto; border-spacing: 0 20px; padding-left: 2%; padding-right: 2%; padding-bottom: 1%;'> 
>                         <tr style='text-align: center;'> 
>                             <td colspan='2'> 
>                                 <span>
>                                     <h1 id='Title' style='color:#FFFFFF;'><YOUR NAME> Jellyfin Newsletter</h1>
>                                     <h3 id='Date' style='color:#FFFFFF;'>2024</h3>
>                                 </span> 
>                             </td> 
>                         </tr> 
>                          
>                         {EntryData}
>                          
>                     </table> 
>                     <br>
>                     <br>
>                     <span style="text-align: center; display: block;">
>                         <a href="<YOUR DOMAIN>" style="border: 2px solid white; border-radius: 15px; padding: 10px; background: linear-gradient(to bottom right, #713a91, #0785b6); color: white; text-decoration: none;"><SOME TEXT></a>
>                     </span>
>                     <br>
>                     <br>
>                 </td> 
>             </tr> 
>         </table> 
>     </div> 
> </html>
> ```
> 
> Entry Data:
> 
> ```
> <tr class='boxed' style='outline: 2px solid #D3D3D3; background-color: #1f1f1f; border-radius: 20px; overflow: hidden;'>
>                             <td class='lefttable' style='padding-right: 5%; padding-left: 2%; padding-top: 2%; padding-bottom: 2%; '>
>                                 <img style='width: 200px; height: 300px; border-radius: 20px;' src='{ImageURL}'>
>                             </td>
>                             <td class='righttable' style='vertical-align: top; padding-left: 5%; padding-right: 2%; padding-top: 2%; padding-bottom: 2%;'>
>                                 <p>
>                                     <div style="border: 2px solid white; border-radius: 20px; padding: 15px;">
>                                         <div id='SeriesTitle' class='text' style='color: #FFFFFF; text-align: center;'><h3>{Title}</h3></div>
>                                         <div id='SeasonEpisode' class='text' style='color: #FFFFFF;'>{SeasonEpsInfo}</div>
>                                         <hr>
>                                         <div id='Description' class='text' style='color: #FFFFFF;'>{SeriesOverview}</div>
>                                     </div>
>                                 </p>
>                             </td>
>                         </tr>
> ```
> 
> Don't forget to insert your custom data in the body.

Have you considered submitting your improvements back to that dev? Looks like they would appreciate it

**Comment by BLACK4585** [comment]
Yes I will do this, but a few improvements have to be done first. The layout is a bit hacky rn.

**Comment by herkalurk** [comment]
Wondering if this is still something that could be added,  I'm using plex,  am looking to move from ombi and have users who like the newsletter.

**Comment by InactiveImmortal** [comment]
> Subscribing for updates on this. The Jellyfin Newsletter plugin is woefully out of date, and I can't send emaills with it wiht my email provider.

Same here!

**Comment by PartyingChair** [comment]
Would like to add my voice to this feature request. Especially as an Emby user, I cannot configure the body of the notification e-mail at all, so the solution mentioned here does not work for me. Hopefully Seerr can add something like it, I think it would fit well.

Thank you

**Comment by thickconfusion** [comment]
I'm still running Ombi _just_ to have it send out the newsletter... Please add, thank you!

**Comment by diedrichg** [comment]
I too would love to have this and was very surprised to find out that it wasn't a feature. I'm tempted to go back to Ombi _just_ for this feature.

**Comment by herkalurk** [comment]
> I'm still running Ombi _just_ to have it send out the newsletter... Please add, thank you!

If you have tautulli that has a newsletter engine as well, and can import your plex users email natively.  I came from OMBI as well and already had tautulli.

**Comment by DereckRadford** [comment]
Is this something that is a possibility of being implemented, considering it's been open since 2022, so far, I have enjoyed Seerr, but this is the only big detractor for me so far.
